packages/framework/src/Model/Customer/DeliveryAddress.php

```diff
@@ -126,41 +126,41 @@ public function getCompanyName()
     }
 
     /**
-     * @return string|null
+     * @return string
      */
-    public function getFirstName()
+    public function getFirstName(): string
     {
         return $this->firstName;
     }
 
     /**
-     * @return string|null
+     * @return string
      */
-    public function getLastName()
+    public function getLastName(): string
     {
         return $this->lastName;
     }
 
     /**
-     * @return string|null
+     * @return string
      */
-    public function getStreet()
+    public function getStreet(): string
```

Comment:
return typehints on getters for ORM annotated fields are forbidden in favor of better extensibility. 
Check phpstan to see issues, please

Reply:
I did not notice, sorry. Fixed https://github.com/shopsys/shopsys/pull/2494/commits/6e6c837fc1e9a5871ce2038992c83e97828d094f